packages/styling/src/styles/theme.ts

```diff
@@ -1,33 +1,48 @@
-import { Customizations, merge } from '@uifabric/utilities';
+import { Customizations, merge, getWindow } from '@uifabric/utilities';
 import { IPalette, ISemanticColors, ITheme, IPartialTheme, IFontStyles } from '../interfaces/index';
 import { DefaultFontStyles } from './DefaultFontStyles';
 import { DefaultPalette } from './DefaultPalette';
 import { DefaultSpacing } from './DefaultSpacing';
 import { loadTheme as legacyLoadTheme } from '@microsoft/load-themed-styles';
 import { DefaultEffects } from './DefaultEffects';
+import { registerDefaultFontFaces, getFontBaseUrl } from '../styles/DefaultFontStyles';
 
-let _theme: ITheme = createTheme({
-  palette: DefaultPalette,
-  semanticColors: _makeSemanticColorsFromPalette(DefaultPalette, false, false),
-  fonts: DefaultFontStyles,
-  isInverted: false,
-  disableGlobalClassNames: false
-});
-let _onThemeChangeCallbacks: Array<(theme: ITheme) => void> = [];
+let _hasInitialized = false;
+let _theme: ITheme;
+const _onThemeChangeCallbacks: Array<(theme: ITheme) => void> = [];
 
 export const ThemeSettingName = 'theme';
 
-if (!Customizations.getSettings([ThemeSettingName]).theme) {
-  let win = typeof window !== 'undefined' ? window : undefined;
+/**
+ * Initializes the default theme. Important to call from any code which assumes
+ * a default theme object will be available from Customization global settings.
+ */
+export function initializeDefaultTheme(): void {
+  if (!_hasInitialized) {
+    _hasInitialized = true;
+    registerDefaultFontFaces(getFontBaseUrl());
+
+    _theme = Customizations.getSettings([ThemeSettingName]).theme;
+
+    if (!_theme) {
+      const win = getWindow();
+
+      // tslint:disable:no-string-literal no-any
+      if (win && (win as any)['FabricConfig'] && (win as any)['FabricConfig'].theme) {
+        _theme = createTheme((win as any)['FabricConfig'].theme);
+      } else {
+        _theme = createTheme({
+          palette: DefaultPalette,
```

Comment:
why is this necessary? doesn't createTheme() already populate all the defaults for you?

Reply:
This takes the initialization into something that can be explicitly imported.

Reply:
@phkuo, @dzearing had a response to your question. Could you please give this another look and review?